tests/RobotFramework/tests/cumulocity/benchmarks/benchmarks.robot

```diff
@@ -3,7 +3,7 @@ Resource    ../../../resources/common.resource
 Library    Cumulocity
 Library    ThinEdgeIO
 
-Test Tags    theme:c8y    theme:benchmarks    \#2326
+Force Tags    theme:c8y    theme:benchmarks    \#2326
```

Comment:
I'm no Robot test framework expert, but as per the [docs](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#deprecation-of-force-tags-and-default-tags) `Force Tags` seems to be deprecated and `Test Tags` seems to be the new norm.